.changeset/fluffy-readers-add.md

```diff
@@ -0,0 +1,6 @@
+---
+"@astrojs/internal-helpers": minor
+"astro": minor
+---
+
+Add multiple CDN option, `build.assetsPrefix` also add object type for can config multiple CDN assets prefix.
```

Comment:
```suggestion
Adds the option to pass an object to `build.assetsPrefix`. This allows for the use of multiple CDN prefixes based on the target filetype.
```
Great first starting changeset!

The changesets in Astro usually follow the following format:
`this PR` \<verb>s `...`
This is just to ensure consistency in the changesets!
Thank you for writing this well to start out with! :)

Reply:
Thanks I will solve it